pkgs/tools/misc/fontfor/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, pkg-config
+, expat
+, freetype
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fontfor";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "7sDream";
+    repo = "fontfor";
+    rev = "v${version}";
+    sha256 = "1b07hd41blwsnb91vh2ax9zigm4lh8n0i5man0cjmxhavvbfy12b";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  buildInputs = [
+    expat
+    freetype
+  ];
```

Comment:
Upstream is [mentioning](https://github.com/7sDream/fontfor#dependencies) [`fontconfig`](https://search.nixos.org/packages?channel=unstable&show=fontconfig&query=fontconfig) as a requirement.

Not sure if that's the cause for the runtime warnings:

```bash
...ew/pr-113143]$ ./results/fontfor/bin/fontfor U+05d0
Fontconfig warning: line 5: unknown element "description"
Fontconfig error: line 5: invalid attribute 'domain'
Fontconfig warning: "/etc/fonts/conf.d/10-hinting-slight.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/20-unhint-small-vera.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/30-metric-aliases.conf", line 4: unknown element "description"
[...]
```

Reply:
I don't get these warnings whatsoever but it seems like it links to libfontconfig when I provide it to buildInputs.